_docs_integrate/attribute-introduction.md

```diff
@@ -21,3 +21,132 @@ required_by:
+
+Please note that when creating a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) that contains a complex IdentityAttribute in its `content` property, additional LocalAttributes are only created automatically if the LocalAttribute is a RepositoryAttribute. However, if the LocalAttribute is an own shared IdentityAttribute or a peer shared IdentityAttribute, no additional LocalAttributes are created. More details on the terminology relating to [LocalAttributes and IdentityAttributes]({% link _docs_integrate/attribute-introduction.md %}#localattributes-and-identityattributes) can be found in the next section.
+{: .notice--info}
+
+### LocalAttributes and IdentityAttributes
+
+From a technical perspective, an IdentityAttribute is always stored as the `content` of a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute). Depending on the values of certain properties of the LocalAttribute, a LocalAttribute whose `content` is given by an IdentityAttribute is also referred to as a **RepositoryAttribute**, an **own shared IdentityAttribute** or a **peer shared IdentityAttribute**.
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/62b3b352-a6d7-4826-8693-e8527b0370e4" id="zVGxoOkr.UEQ"></iframe></div>
```

Comment:
- I see your point! I can also see how this could be realised for the IdentityAttribute box. What about the RelationshipAttributes box? There, "you" has RelationshipAttributes that belong to both them and the other party, which means that `owner` doesn't quite fit instead. It would be important to me to have a consistent labelling of the parties in the boxes. Any suggestions? (E.g. try to split the RelationshipAttribute diagramm appropriately like proposed by you to me below?)
- As far as I know, the boxes all have the same width (640px). Did you actually mean the different heights, or is there a formatting setting for the width that I didn't notice?

Reply:
- I agree, it would be nice to have it consistent with the following diagram(s). I think the best way in fact is to split the diagram for RealtionshipAttributes. Not only does this improve its comprehensibility, but it would also allow us to declare the owner of an Attribute unambiguously.
- I'll re-check the Attribute boxes and possibly contact you directly.